config/deployments/integration_test.toml

```diff
@@ -0,0 +1,285 @@
+[log.file]
+enabled = false
```

Comment:
Do we need this here, considering that this has also been included in the `env_specific.toml` file?